proposals/devcontainer-collections.md

```diff
@@ -0,0 +1,110 @@
+# Proposal
```

Comment:
Am I missing the reference the concept of centralized 'index(es)'.  

RIght now I have this flat file with a single reference to a single collection: https://github.com/joshspicer/devcontainer-index

We would then add pointers here to collections we "endorse".  Others in the community would be able to produce their own indexes, which point whatever set of collections they would like.  Additional indexes could be added to Codespaces via org policy, or via vscode by a setting.

Reply:
I see the indexes as more of a client feature, where clients define how to store and search it. Also, how users would add a reference to a collection to search their internal features for example.

Reply:
Is that where we would have the 3 collections from Codespaces, VS Code and the community to start with?

Reply:
Yea, I want us to publish an index file that will point to our three collections.  All we need to do is hardcode a single index file, and _that_ will point to the `N` collections we want to pull from by default.